service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1928,3 +1932,17 @@ components:
           type: array
           items:
             type: string
+
+  securitySchemes:
```

Comment:
Is this applicable to IAP or just bearer tokens?

Reply:
Good question. I think this OpenAPI definition only matters for our local server right now (in particular hitting the Swagger UI page, which I do often during development), and may matter in the future if another service uses our generated client library. I don't think it makes a difference in our dev deployment, though I have not tested that.